README.rst

```diff
@@ -145,20 +145,20 @@ This is not an oversight, it's a decision with the following rationale:
 1. Polling is "fast enough", particularly since PEP 471 introduced fast ``scandir``.
 
    With a reasonably large project like the TutorCruncher code base with 850 files and 300k lines
-   of code *watchgod* can scan the entire tree in ~24ms. With a scan interval of 400ms that's roughly
+   of code *watchgod* can scan the entire tree in ~24ms. The scan interval is 400ms which is roughly
    5% of one CPU - perfectly acceptable load during development.
 
 2. The clue is in the title, there are at least 4 different file notification systems to integrate
-   with, most of them not trivial. And that's before we get to changes between different OS versions.
+   with, most of them not trivial. That is all before we get to changes between different OS versions.
```

Comment:
This doesn't look necessary the previous grammar, while informal is fine IMHO.

Reply:
Sorry about that, I have to make minor github documentation edits for my technical writing course so I have been looking for repositories to edit. Did not mean to offend at all.

Reply:
okay, no problem.

Reply:
If you're happy to make the changes I suggested, I'll merge this.

Reply:
If you're looking for documentation changes, have a look at [this](https://github.com/samuelcolvin/pydantic/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation) list of issues.

I'd willingly review PRs for any of those.

Reply:
Thanks, doing them right now! :)

Reply:
Changes have been made as requested. They are part of the second commit. Thank you for providing additional documentation to work on. I will look into them soon.